From 753fc2d26af779a9c0508b86578c5644da4f4a8d Mon Sep 17 00:00:00 2001
From: "Andrew F. Davis" <afd@ti.com>
Date: Thu, 31 Mar 2016 00:32:49 +0000
Subject: [PATCH 44/46] TEMP: net: prueth: Do not stop kernel TX queue

We do not currently have a method to detect when the hardware TX queue
is no longer full and so we never re-start the kernel TX queue, without
this, stopping the queue will lock the interface. Remove this stop here.

Signed-off-by: Andrew F. Davis <afd@ti.com>
Tested-by: Jason Reeder <jreeder@ti.com>
---
 drivers/net/ethernet/ti/ti-prueth.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/net/ethernet/ti/ti-prueth.c b/drivers/net/ethernet/ti/ti-prueth.c
index c84b3f7..545fa4c 100644
--- a/drivers/net/ethernet/ti/ti-prueth.c
+++ b/drivers/net/ethernet/ti/ti-prueth.c
@@ -1244,7 +1244,6 @@ static int emac_ndo_start_xmit(struct sk_buff *skb, struct net_device *ndev)
 fail_tx:
 	/* error */
 	ndev->stats.tx_dropped++;
-	netif_stop_queue(ndev);
 
 	return NETDEV_TX_BUSY;
 }
-- 
1.9.1

